script.js

```diff
@@ -0,0 +1,110 @@
+}
+
+function getBooks() {
+  // localStorage.clear();
+  const tempStorage = JSON.parse(localStorage.getItem('awesomeBooks'));
+  if (tempStorage) {
+    awesomeBooks = tempStorage;
+  }
+}
+
+getBooks();
+
+function getBookId() {
+  let tempBoy = 0;
+  try {
+    tempBoy = awesomeBooks[awesomeBooks.length - 1].bookId + 1;
+  } catch {
+    tempBoy = 0;
+  } finally {
+    bookIncrement = tempBoy;
+  }
+}
+getBookId();
+
+function deleteBook(bookNum) {
+  const container = document.getElementById(`book${bookNum}`);
+  container.remove();
+  awesomeBooks = awesomeBooks.filter((book) => book.bookId !== bookNum);
+  localStorage.clear();
+  localStorage.setItem('awesomeBooks', JSON.stringify(awesomeBooks));
+}
+
+function displayBook(...args) {
+  const tempArray = ['Title', 'Author'];
+  const mainContainer = document.getElementById('BookContainer');
+  const container = document.createElement('div');
+  container.id = `book${args[2]}`;
+  container.classList.add(
+    'container',
+    'mx-auto',
+    'border',
+    'rounded',
+    'justify-content-center',
+    'my-2',
+  );
+  for (let i = 0; i < tempArray.length; i += 1) {
+    const titleContainer = document.createElement('div');
+    titleContainer.classList.add(
+      'col-md-8',
+      'text-center',
+      'justify-content-center',
+      'mx-auto',
+      'my-2',
+      'd-flex',
+    );
+    const titleLabel = document.createElement('h3');
+    titleLabel.classList.add('h3', 'mx-2');
+    titleLabel.textContent = `${tempArray[i]}:`;
+    titleContainer.appendChild(titleLabel);
+    const bookTitle = document.createElement('h2');
+    bookTitle.classList.add('h2', 'mx-2');
+    bookTitle.innerText = args[i];
+    titleContainer.appendChild(bookTitle);
+    container.appendChild(titleContainer);
+  }
+  const removeButton = document.createElement('button');
+  const [, , selectedBook] = args;
+  removeButton.id = selectedBook;
+  removeButton.classList.add('btn', 'btn-primary', 'col-5', 'my-2');
+  removeButton.innerText = 'Remove Book';
+  container.appendChild(removeButton);
+  mainContainer.appendChild(container);
+
+  removeButton.addEventListener('click', () => {
+    deleteBook(selectedBook);
+  });
+}
+
+function addBook() {
+  const bookTitleInput = document.getElementById('BookTitle').value;
+  const bookAuthorInput = document.getElementById('BookAuthor').value;
+
+  awesomeBooks.push(new Book(bookTitleInput, bookAuthorInput, bookIncrement));
+
+  localStorage.setItem('awesomeBooks', JSON.stringify(awesomeBooks));
+
+  displayBook(bookTitleInput, bookAuthorInput, bookIncrement);
+  bookIncrement += 1;
+}
+
+document.getElementById('SubmitButton').addEventListener('click', addBook);
+function printStorage() {
+  if (!awesomeBooks.empty) {
+    for (let i = 0; i < awesomeBooks.length; i += 1) {
+      displayBook(
+        awesomeBooks[i].bookTitle,
+        awesomeBooks[i].bookAuthor,
+        awesomeBooks[i].bookId,
+      );
+    }
+  }
+}
+
+printStorage();
```

Comment:
- [ ] You've done pretty well with the way you've structured your syntax and logic :+1: 
However, it is always better to program our script in such a way that we allow all the static contents of the DOM to be completely loaded before rendering our dynamic contents. You may have something like: 

 `document.addEventListener('DOMContentLoaded', printStorage()); `
 
 Optional: Please note that the use of `DOMContentLoaded` is preferred over `load`. A different event, `load`, should be used only to detect a fully loaded page. It is a common mistake to use load where DOMContentLoaded would be more appropriate.", see https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event